CHANGELOG.md

```diff
@@ -5,13 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.5.1] - 2024-08-01
 
 #### Added
  - Added usage example for an ElligatorSwift key exchange.
 
+#### Changed
+ - The default size of the precomputed table for signing was changed from 22 KiB to 86 KiB. The size can be changed with the configure option `--ecmult-gen-kb` (`ECMULT_GEN_KB` for CMake).
```

Comment:
```suggestion
 - The default size of the precomputed table for signing was changed from 22 KiB to 86 KiB. The size can be changed with the configure option `--ecmult-gen-kb` (`SECP256K1_ECMULT_GEN_KB` for CMake).
```

Reply:
Does  that mean the entry "(`ECMULT_GEN_KB` for CMake)" for 0.5.0 is wrong (or at least misleading)?

Reply:
I suppose it is misleading.

Reply:
Thanks, fixed. Also added a commit to clarify the 0.5.0 entry.